app/assets/stylesheets/_commons.css.scss

```diff
@@ -1,5 +1,5 @@
 body {
-  background: $white !important;
+  background: white !important;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>!important should not be used